aspnetcore/fundamentals/http-requests/samples/3.x/HttpRequestsSample/Startup.cs

```diff
@@ -25,24 +26,25 @@ public void ConfigureServices(IServiceCollection services)
                 // For sample purposes, assume TodoClient is used in the context of an incoming request.
                 var httpRequest = sp.GetRequiredService<IHttpContextAccessor>().HttpContext.Request;
 
-                httpClient.BaseAddress = new Uri(UriHelper.BuildAbsolute(httpRequest.Scheme, httpRequest.Host, httpRequest.PathBase));
+                httpClient.BaseAddress = new Uri(UriHelper.BuildAbsolute(httpRequest.Scheme,
```

Comment:
```suggestion
                // For sample purposes, assume TodoClient is used in the context of an incoming request.
                httpClient.BaseAddress = new Uri(UriHelper.BuildAbsolute(httpRequest.Scheme,
```

With this code being shown in a snippet, I think the comment should live closer to this line.